ansible/playbooks/rook.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: repository
```

Comment:
Could you explain or even add a comment why repository's facts are gathered?

Reply:
Comment added.